website/app/styles/doc-components/algolia-search/modal.scss

```diff
@@ -0,0 +1,250 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+// ALGOLIA AUTOCOMPLETE SEARCH > MODAL
+
+@use "../../breakpoints" as breakpoint;
+@use "../../typography/mixins" as *;
+
+// This class is applied to the `<body>` to remove the scroll when the modal is opened
+.aa-Detached {
+  height: 100vh;
+  overflow: hidden;
+}
+
+
+// OVERLAY (used to apply an opaque background and _contain_ the modal container)
+
+.aa-DetachedOverlay {
+  position: fixed;
+  top: 0;
+  right: 0;
+  left: 0;
+  z-index: calc(9999 - 1);
+  height: 100vh;
+  margin: 0;
+  padding: 0;
+  background-color: rgba(#000, 0.5);
+}
+
+// MODAL (container)
+
+.aa-DetachedContainer {
+  position: fixed;
+  top: 0;
+  right: 0;
+  left: 0;
+  z-index: 9999;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  max-height: 100vh;
+  margin: 0;
+  overflow: hidden;
+  background: var(--doc-color-white);
+
+  @include breakpoint.with-fixed-sidebar () {
+    position: absolute;
+    top: 90px;
```

Comment:
If this `top: 90px;` value is based partly on the height of the doc-page-header would it make sense to use a variable or else add a comment explaining?

Reply:
It's kinda base, but not exactly. It's a number that "looks right" (visually, and numerically).